src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -132,6 +132,13 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
             }
             else
             {
+                string systemPath = Path.Combine(_rootScriptPath, ScriptConstants.AzureFunctionsSystemDirectoryName);
+
+                if (isDotnetIsolatedApp && !Directory.Exists(systemPath))
+                {
+                    _logger.NoAzureFunctionsFolder(systemPath);
```

Comment:
Will this get logged during placeholder mode?

Reply:
Yes, good catch. Added a check to avoid logging in placeholder mode.